maquetado/scss/_tech.scss

```diff
@@ -0,0 +1,45 @@
+.tech {
+  align-items: center;
+  background-color: $green;
+  border-radius: 5px;
+  display: inline-flex;
+  flex-direction: column;
+  margin: $tech-decoration-offset 0 0 $tech-decoration-offset;
+  max-width: 240px;
+  padding: 2em;
```

Comment:
In Wolox are working at this moment only with pixels 😞 . I hate use pixels but i can't do nothing